<launch>
    <node pkg="vision_lane_detection" type="lane_marker" name="lane_marker">
		<param name="arrow_model_path" value="$(find vision_lane_detection)/data_and_model/arrow_20130308.model"/>
    	<param name="arrow_scale_path" value="$(find vision_lane_detection)/data_and_model/arrow_20130308.range"/>
    	<param name="image_folder_path" value="/home/baoxing/images"/>
    	<param name="extract_training_image" value="false"/>
    	<param name="scale" value="20.0"/>
	</node>
	
</launch>
